content/sensu-go/5.10/dashboard/filtering.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Dashboard filtering"
+linkTitle: "Filtering"
+description: "The Sensu dashboard supports filtering on the events, entities, checks, handlers, and silences pages. Read the doc to learn more."
+version: "5.10"
+product: "Sensu Go"
+platformContent: false
+menu:
+  sensu-go-5.10:
+    parent: dashboard
+---
+
+The Sensu dashboard provides drop-down filters to build customized views of Sensu resources.
+When applied, filters create a unique link, so you can bookmark and share your favorite filter combinations.
+
+- On the **events page**, filter by entity, check, status, and silenced/unsilenced. You can also sort events by severity, last OK, newest, and oldest.
+- On the **entities page**, filter by entity class and subscription.
+- On the **checks page**, filter by subscription and published/unpublished.
+- On the **handlers page**, filter by handler type.
+- On the **silences page**, filter by check and subscription. You can also sort silences by start date.
+
+## Advanced filtering
+
+**LICENSED TIER**: Unlock advanced filtering in the Sensu Go dashboard with a Sensu license. To activate your license, see the [getting started guide][1].
+
+Sensu supports advanced dashboard filtering using the filter bar.
+Select the filter bar and start building custom views using suggested attributes and values, including custom labels.
+
+### Label selectors
+
+To filter by custom labels, use the `labelSelector` filter and the `==` operator.
+For example, to display only resources with a `region: us-west-1` label:
+
+{{< highlight text >}}
+labelSelector: region == "us-west-1"
+{{< /highlight >}}
```

Comment:
@wesleycyu Looking at the events page, I'm trying to filter by entities with a particular label using this syntax, but it doesn't seem to be working. Can you confirm the syntax for filtering events by entity label?

Reply:
@apaskulin It's currently not possible to use the `labelSelector` to filter events by label via entity or check. We don't copy the labels associated with an Entity or Check on to the Event type ([src](https://github.com/sensu/sensu-go/blob/master/agent/check_handler.go#L105-L117)). 

Since Events don't have labels associated with them (events have labels via an Entity or Check), the `labelSelector` on the events page actually does nothing. This was an oversight on our part. 

As a side note: The following fields can be queried on the Events page using the `fieldSelector` ([src](https://github.com/sensu/sensu-go/blob/master/api/core/v2/event.go#L367-L382)):
```
"event.name": String,
"event.namespace": String
"event.check.handlers": Array[String],
"event.check.publish": Boolean,
"event.check.round_robin": Boolean,
"event.check.runtime_assets": Array[String],
"event.check.status": Int,
"event.check.subscriptions": Array[String],
"event.entity.deregister": Boolean,
"event.entity.entity_class": String,
"event.entity.subscriptions": Array[String],
```

Reply:
cc @jamesdphillips

Reply:
Thanks! Updated to reflect this.